src/Text/Microstache/Type.hs

```diff
@@ -109,20 +113,42 @@ instance NFData PName
 data MustacheException
   = MustacheParserException ParseError
     -- ^ Template parser has failed. This contains the parse error.
-    --
-    -- /Before version 0.2.0 it was called 'MustacheException'./
   | MustacheRenderException PName Key
     -- ^ A referenced value was not provided. The exception provides info
     -- about partial in which the issue happened 'PName' and name of the
     -- missing key 'Key'.
   deriving (Eq, Show, Typeable, Generic)
 
-#if MIN_VERSION_base(4,8,0)
-instance Exception MustacheException where
-  displayException (MustacheParserException e) = show e
-  displayException (MustacheRenderException pname key) =
+{-# DEPRECATED MustacheRenderException "Not thrown anymore, will be removed in the next major version of microstache" #-}
```

Comment:
I'm a bit confused by this deprecation for `MustacheRenderException`. Should it actually be `MustacheException` instead?